playbooks/ansible-cloud/k8s/tox.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- hosts: all
+  tasks:
+    - name: Setup ensure-tox role
+      include_role:
+        name: ensure-tox
+
+    - name: Install SELinux Python bindings using yum
```

Comment:
This binary depends on a specific version on Python and are likely to create trouble if we change this version later.

Most of the time we don't use any SELinux specific feature and the tests run well without it. In this case an alternative is to just mock selinux.
https://pypi.org/project/selinux-please-lie-to-me/

Reply:
Without SElinux binding, I saw failures.

Reply:
It should be find if you replace selinux with this package. BTW you can now remove  playbooks/ansible-cloud/k8s/tox.yaml from the PR :-).